general/src/protocol/contexts.rs

```diff
@@ -347,11 +353,137 @@ pub struct TraceContext {
     #[metastructure(max_chars = "enumlike")]
     pub op: Annotated<OperationType>,
 
+    /// Whether the trace failed or succeeded. Currently only used to indicate status of individual
+    /// transactions.
+    pub status: Annotated<TraceStatus>,
+
     /// Additional arbitrary fields for forwards compatibility.
     #[metastructure(additional_properties, retain = "true")]
     pub other: Object<Value>,
 }
 
+/// Trace status
+///
+/// https://github.com/open-telemetry/opentelemetry-specification/blob/8fb6c14e4709e75a9aaa64b0dbbdf02a6067682a/specification/data-http.md#status
```

Comment:
The canonical list of trace statuses is from GRPC: https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/api-tracing.md#statuscanonicalcode

Reply:
Fixed, thanks for catching this